packages/web-api/src/methods.ts

```diff
@@ -62,48 +62,95 @@ export interface AdminAppsApproveArguments extends WebAPICallOptions, TokenOverr
   request_id?: string;
   team_id?: string;
 }
+export interface AdminAppsApprovedListArguments extends WebAPICallOptions, TokenOverridable, CursorPaginationEnabled {
+  team_id?: string;
+  enterprise_id?: string;
+}
+cursorPaginationEnabledMethods.add('admin.apps.approved.list');
 export interface AdminAppsRequestsListArguments extends WebAPICallOptions, TokenOverridable, CursorPaginationEnabled {
   team_id?: string;
 }
+cursorPaginationEnabledMethods.add('admin.apps.requests.list');
 export interface AdminAppsRestrictArguments extends WebAPICallOptions, TokenOverridable {
   app_id?: string;
   request_id?: string;
   team_id?: string;
 }
+export interface AdminAppsRestrictedListArguments extends WebAPICallOptions, TokenOverridable, CursorPaginationEnabled {
+  team_id?: string;
+  enterprise_id?: string;
+}
+cursorPaginationEnabledMethods.add('admin.apps.restricted.list');
+export interface AdminConversationsSetTeamsArguments extends WebAPICallOptions, TokenOverridable {
+  channel_id: string;
+  team_id?: string;
+  target_team_ids?: string[];
```

Comment:
Unfortunately, this one needs to be `string` as it's a comma-separated list of string values. Putting a comment saying `// comma-separated list of team_id` as with others like `ConversationsInviteArguments#users` etc.

Reply:
Hey @seratch, according to https://slack-github.com/slack/webapp/blob/master/api_metadata/methods/admin.conversations/setTeams.api#L34, this one is an array of type strings

Reply:
Verified this should be an array of strings. This should be updated in the documentation.

Reply:
Thanks @shaydewael! I'm going to merge this one in now

Reply:
Oh nvm, I have to wait until @seratch approves before I can merge.